include/secp256k1.h

```diff
@@ -119,6 +119,20 @@ typedef int (*secp256k1_nonce_function)(
 #  define SECP256K1_INLINE inline
 # endif
 
+#ifndef SECP256K1_API
+# if defined(_WIN32)
+#  ifdef SECP256K1_BUILD
```

Comment:
Can you rename `SECP256K1_BUILD` to `SECP256K1_EXPORTS`? It is weird that `SECP256K1_BUILD` is defined for dynamic builds but not for static builds.